fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -663,6 +685,10 @@ struct PhysicalShard {
 	double deleteTimeSec;
 };
 
+bool DataShard::initialized() const {
```

Comment:
Usually the physical shard is ready when we access the data. Could you give examples when the physical shard could be uninitialized?

Reply:
I don't remember the exact reason I added this, it is likely related to a failed restore operation. That has been improved, I removed this check and it passed all the sim-tests.